test/CMakeLists.txt

```diff
@@ -56,9 +57,13 @@ if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID MATCHES "GNU"
         set(CMAKE_C_COMPILER_TARGET arm-linux-gnueabi)
         set(CMAKE_CXX_COMPILER_TARGET arm-linux-gnueabi)
 
-        include_directories(/usr/${ARM_ARCH_DIRECTORY}/include/c++/${ARM_GCC_VER}/${ARM_ARCH_DIRECTORY}/)
-        include_directories(/usr/${ARM_ARCH_DIRECTORY}/include/c++/${ARM_GCC_VER}/)
-        include_directories(/usr/${ARM_ARCH_DIRECTORY}/include/)
+        set(ARM_ARCH_DIRECTORY "arm-linux-gnueabi" CACHE STRING "ARM arch header dir")
+        set(ARM_GCC_VER "4.7.3" CACHE STRING "ARM GCC header dir")
+        if (CMAKE_CXX_COMPILER_ID MATCHES "GNU")
```

Comment:
Why this extra check? Cross compiling from Clang still requires a GNU cross sysroot :-)

Reply:
I'll remove it.

Reply:
Done, thanks for the check!